SafeExamBrowser.SystemComponents/VirtualMachineDetector.cs

```diff
@@ -97,69 +106,114 @@ private bool IsVirtualRegistry()
 		{
 			var isVirtualMachine = false;
 
-			// check historic hardware profiles
-			var hardwareConfKey = Microsoft.Win32.Registry.LocalMachine.OpenSubKey("SYSTEM\\HardwareConfig");
-			if (hardwareConfKey != null)
+			// the resulting IsVirtualRegistry() would be massive so split it
+			isVirtualMachine |= IsVirtualRegistryHardwareConfig();
+			isVirtualMachine |= IsVirtualRegistryDeviceCache();
+
+			return isVirtualMachine;
+		}
+
+
+		/// <summary>
```

Comment:
Dito (comment). The comment however appears to indicate resp. contain information which isn't quite the same as the method name, hence the question would be whether the method name is meaningful this way or if it could be improved or should be renamed to e.g. `HasHistoricVirtualMachineHardwareConfiguration()`?